NOAA-FIMS/FIMS
# refactor: Deletes issue and PR templates

# What is the feature?
* Moves PR and issue templates to NOAA-FIMS/.github/.github so they are available to all repositories within NOAA-FIMS.

# How have you implemented the solution?
Moved the files to NOAA-FIMS/.github/.github see the commit message for the exact commit.

# Does the PR impact any other area of the project?
The refactor issue template was not moved.

# How to test this change
Go to another repository in FIMS and see the available issue templates 🤯 or see below
![image](https://github.com/NOAA-FIMS/FIMS/assets/4108564/750fb61c-6aa3-49ee-81de-caf7251408ce)


# Developer pre-PR checklist
